ansible/configs/ocp4-cluster/destroy_env_gcp.yml

```diff
@@ -41,6 +41,14 @@
       set_fact:
         ansible_ssh_extra_args: "{{ ansible_ssh_extra_args|default() }} -F {{ hostvars['localhost'].ansible_ssh_config }}"
 
+- name: Start all GCP instances if they are stopped
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tasks:
+    - include_tasks: gcp_instances_start.yml
+
```

Comment:
I can't see a lifecycle playbook in ansible/ for gcp.
Why not add one and then use it from here?